src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -189,6 +189,209 @@ public static class ILoggerExtensions
             logger.LogWarning(RequestFormat, new RequestLogEntry(request.Method.ToString(), host, resourcePath, operationName, statusCode, duration, context));
         }
 
+        /// <summary>
+        ///     Logs an Azure Service Bus topic request.
+        /// </summary>
+        /// <param name="logger">The logger instance to track the telemetry.</param>
+        /// <param name="serviceBusNamespace">The namespace where the Azure Service Bus topic is registered.</param>
+        /// <param name="topicName">The name of the Azure Service Bus topic.</param>
+        /// <param name="operationName">The name of the operation of the request.</param>
+        /// <param name="isSuccessful">The indication whether or not the Azure Service Bus topic request was successfully processed.</param>
+        /// <param name="measurement">The instance to measure the latency duration of the Azure Service Bus topic request.</param>
+        /// <param name="context">The telemetry context that provides more insights on the Azure Service Bus topic request.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> or the <paramref name="measurement"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="serviceBusNamespace"/> or <paramref name="topicName"/> is blank.</exception>
+        public static void LogServiceBusTopicRequest(
+            this ILogger logger,
+            string serviceBusNamespace,
+            string topicName,
+            string operationName,
+            bool isSuccessful,
+            DependencyMeasurement measurement,
+            Dictionary<string, object> context = null)
+        {
+            Guard.NotNull(logger, nameof(logger), "Requires an logger instance to track telemetry");
+            Guard.NotNullOrWhitespace(serviceBusNamespace, nameof(serviceBusNamespace), "Requires an Azure Service Bus namespace to track the topic request");
+            Guard.NotNullOrWhitespace(topicName, nameof(topicName), "Requires an Azure Service Bus topic name to track the topic request");
+            Guard.NotNull(measurement, nameof(measurement), "Requires an instance to measure the Azure Service Bus topic request process latency duration");
+
+            // TODO: the dependency date from the measurement instance is not used when tracking the request.
```

Comment:
Must something be done with this todo before completing the PR ?

Reply:
Not sure, was something I came across. We don't use the data of the measurement, only the tracked time. It may not be an issue here, though.

Reply:
I guess we'll need to build a preview package , call this from a test project and see how it is displayed in Application Insights then ?

Reply:
We can already do that in the integration tests, but this PR doesn't yet do the link with Application Insights (see description), so we can't test this yet.

Reply:
I don't think this will have issues as the dependency data is used for dependency tracking, but not sure. Maybe we should include the data in the telemetry context.

Reply:
I'll add an issue for this and close this conv.